### PR TITLE
reset: add page

### PR DESCRIPTION
**Pull request by sbrl**
The reset command comes with many more options, but most of them are quite advanced and you (almost) never need to use them - I certainly haven't had to yet.

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/reset.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# reset
+
+> Reinitalises the current terminal. Clears the entire terminal screen.
```
Comment (pxgamer) [inline pages/linux/reset.md]: Typo in `Reinitialises`?

File: pages/linux/reset.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- Reinitalise the current terminal:
```
Comment (pxgamer) [inline pages/linux/reset.md]: Same here?

File: pages/linux/reset.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/reset.md]: Oops :P

**Comment by sbrl** [comment]
Thanks for that typo fix - I've always had trouble with the middle of words lol :P

**Pull request merged**